kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -301,10 +302,11 @@ public class BigQuerySinkConfig extends AbstractConfig {
   );
   private static final ConfigDef.Importance MERGE_INTERVAL_MS_IMPORTANCE = ConfigDef.Importance.LOW;
   private static final String MERGE_INTERVAL_MS_DOC =
-      "How often (in milliseconds) to perform a merge flush, if upsert/delete is enabled. Can be "
-      + "set to -1 to disable periodic flushing.";
+      "How often (in milliseconds) to perform a merge flush, if upsert/delete is enabled. Can be set to -1" +
+              " to disable periodic flushing. If enabled should be set to at least 10 seconds(A validation" +
+              " would be introduced in future release to this effect). Either " +MERGE_INTERVAL_MS_CONFIG + " or "
+              + MERGE_RECORDS_THRESHOLD_CONFIG + " must be enabled.";
```

Comment:
How often (in milliseconds) to perform a merge flush, if upsert/delete is enabled. Can be set to -1 to disable periodic flushing. Either MERGE_INTERVAL_MS_CONFIG or MERGE_RECORDS_THRESHOLD_CONFIG, or both must be enabled.
This should not be set to less than 10 seconds. A validation would be introduced in a future release to this effect.

Reply:
<img width="1526" alt="Screenshot 2022-12-28 at 3 34 41 PM" src="https://user-images.githubusercontent.com/117655560/209795528-b2eaf3b4-d0aa-4db4-b3b8-34c6e0a61360.png">